include/common/el3_common_macros.S

```diff
@@ -70,10 +70,12 @@
 	isb
 
 	/* ---------------------------------------------------------------------
-	 * Enable the SError interrupt now that the exception vectors have been
+	 * Enable the Abort interrupt now that the exception vectors have been
```

Comment:
This needs more clarification. We're still enabling the SError interrupt but we're also enabling SError handling at EL3.